Tools/autotest/pysim/util.py

```diff
@@ -329,73 +360,76 @@ def BodyRatesToEarthRates(dcm, gyro):
 
     (phi, theta, psi) = dcm.to_euler()
 
-    phiDot   = p + tan(theta)*(q*sin(phi) + r*cos(phi))
-    thetaDot = q*cos(phi) - r*sin(phi)
+    phiDot   = p + tan(theta) * (q * sin(phi) + r * cos(phi))
+    thetaDot = q * cos(phi) - r * sin(phi)
     if fabs(cos(theta)) < 1.0e-20:
         theta += 1.0e-10
-    psiDot   = (q*sin(phi) + r*cos(phi))/cos(theta)
+    psiDot   = (q * sin(phi) + r * cos(phi)) / cos(theta)
     return Vector3(phiDot, thetaDot, psiDot)
 
-radius_of_earth = 6378100.0 # in meters
+radius_of_earth = 6378100.0  # in meters
+
 
 def gps_newpos(lat, lon, bearing, distance):
-    '''extrapolate latitude/longitude given a heading and distance 
-    thanks to http://www.movable-type.co.uk/scripts/latlong.html
-    '''
+    """Extrapolate latitude/longitude given a heading and distance
+    thanks to http://www.movable-type.co.uk/scripts/latlong.html .
+    """
     from math import sin, asin, cos, atan2, radians, degrees
-    
+
     lat1 = radians(lat)
     lon1 = radians(lon)
     brng = radians(bearing)
-    dr = distance/radius_of_earth
-    
-    lat2 = asin(sin(lat1)*cos(dr) +
-                cos(lat1)*sin(dr)*cos(brng))
-    lon2 = lon1 + atan2(sin(brng)*sin(dr)*cos(lat1), 
-                        cos(dr)-sin(lat1)*sin(lat2))
+    dr = distance / radius_of_earth
+
+    lat2 = asin(sin(lat1) * cos(dr) +
+                cos(lat1) * sin(dr) * cos(brng))
+    lon2 = lon1 + atan2(sin(brng) * sin(dr) * cos(lat1),
+                        cos(dr) - sin(lat1) * sin(lat2))
     return (degrees(lat2), degrees(lon2))
 
 
 def gps_distance(lat1, lon1, lat2, lon2):
-    '''return distance between two points in meters,
+    """Return distance between two points in meters,
     coordinates are in degrees
-    thanks to http://www.movable-type.co.uk/scripts/latlong.html'''
+    thanks to http://www.movable-type.co.uk/scripts/latlong.html ."""
     lat1 = math.radians(lat1)
     lat2 = math.radians(lat2)
     lon1 = math.radians(lon1)
     lon2 = math.radians(lon2)
     dLat = lat2 - lat1
     dLon = lon2 - lon1
 
-    a = math.sin(0.5*dLat)**2 + math.sin(0.5*dLon)**2 * math.cos(lat1) * math.cos(lat2)
-    c = 2.0 * math.atan2(math.sqrt(a), math.sqrt(1.0-a))
+    a = math.sin(0.5 * dLat)**2 + math.sin(0.5 * dLon)**2 * math.cos(lat1) * math.cos(lat2)
+    c = 2.0 * math.atan2(math.sqrt(a), math.sqrt(1.0 - a))
     return radius_of_earth * c
 
+
 def gps_bearing(lat1, lon1, lat2, lon2):
-    '''return bearing between two points in degrees, in range 0-360
-    thanks to http://www.movable-type.co.uk/scripts/latlong.html'''
+    """Return bearing between two points in degrees, in range 0-360
+    thanks to http://www.movable-type.co.uk/scripts/latlong.html ."""
     lat1 = math.radians(lat1)
     lat2 = math.radians(lat2)
     lon1 = math.radians(lon1)
     lon2 = math.radians(lon2)
-    dLat = lat2 - lat1
+    # dLat = lat2 - lat1
```

Comment:
I guess you commented this because it isn't used? Either there is something wrong here and needs to be checked or if really isn't needed then we should remove it.